packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -220,6 +220,7 @@ def __init__(
         log_file: Optional[str] = None,
         env_file: Optional[str] = None,
         logger: logging.Logger = _default_logger,
+        peer_registration_delay: Optional[float] = None,
```

Comment:
we need a better way to pass a configuration dictionary to the `Libp2pNode` that will be used for configuration tuning